packages/components/time-picker/src/time-picker-com/basic-time-spinner.vue

```diff
@@ -233,11 +233,38 @@ const scrollDown = (step: number) => {
   const total = currentScrollbar.value === 'hours' ? 24 : 60
   now = (now + step + total) % total
 
+  if (props.arrowControl) {
+    now = findNextUnDisabled(label, now, step, total)
+  }
   modifyDateField(label, now)
   adjustSpinner(label, now)
   nextTick(() => emitSelectRange(label))
 }
-
+const findNextUnDisabled = (
+  type: TimeUnit,
+  value: number,
+  step: number,
+  total: number
+) => {
+  let res: number = value
```

Comment:
```suggestion
  let res = value
```

Such type declarations can be ignored